sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -5447,6 +5447,26 @@
           "properties": {},
           "lockKey": true,
           "defaultValue": "365"
+        },
+        {
+          "input": true,
+          "tableView": true,
+          "key": "currentTaxYear",
+          "label": "currentTaxYear",
+          "protected": false,
+          "unique": false,
+          "persistent": true,
+          "type": "hidden",
+          "tags": [],
+          "conditional": {
+            "show": "",
+            "when": null,
+            "eq": ""
+          },
+          "properties": {},
+          "customClass": "",
+          "lockKey": true,
+          "calculateValue": "value = data.programYear ? data.programYear: \"\";"
```

Comment:
If `data.programYear` already has the desired value, is there a need to create a second hidden field?

Reply:
About the justification "Just in case if we need know Current tax year is fetched", when that would be a benefit since this is only used to be displayed in some labels and is derived from the program year?